### ISSUE TITLE
[FR] Reverb

### ISSUE THREAD

**Issue by sjaehn** [sjaehn/BOops#7]
Request by @unfa .

**Comment by sjaehn** [comment]
@unfa . I took a look into the MVerb source code. Fine. Or the Faust-created ArtyFx Roomy. Anyway TODO for version 1.4.0.

**Comment by sjaehn** [comment]
Changed to TODO for version 1.6.0

**Comment by sjaehn** [comment]
@unfa , now implemented in the 1.5 branch. I decided to take Ardour's Community Edition a-Reverb as this one is really fast. And seems to be less complicated and less artifacts producing than MVerb. 

At this moment, I only exposed the room size parameter. But there are also other parameters like input gain and feedback that can additionally be exposed (not exposed in a-Reverb). The blend parameter, that is equivalent to mix is already controlled by the plugin engine.

And I keep my eyes open for a really glitchy extreme reverb. Maybe something like Infinity2 by Chris Johnson?
